libs/traits/src/rewards.rs

```diff
@@ -0,0 +1,180 @@
+// Copyright 2022 Centrifuge Foundation (centrifuge.io).
```

Comment:
I guess this file is already in.

Reply:
Yes, this branch was over the already merged branch. I need to rebase, thanks for the reminder!

Reply:
Done